src/firmware/linux/host/types/snp.rs

```diff
@@ -108,7 +108,7 @@ impl CertTableEntry {
     /// ```
     ///
     #[cfg(target_os = "linux")]
-    pub fn uapi_to_vec_bytes(table: &Vec<UAPI::CertTableEntry>) -> Result<Vec<u8>, CertError> {
```

Comment:
@larrydewey 

`clippy` suggested this change. Of course we want to follow its suggestions mostly, but this would break API... drop it for now? It provides no functional change and isn't exactly necessary.

Reply:
I think the change should be fine, but if it is breaking API, maybe make it part of a different PR?